packages/plugins/typescript-resolvers/src/index.ts

```diff
@@ -58,15 +58,39 @@ export interface TypeScriptResolversPluginConfig extends RawResolversConfig {
    * ```
    */
   showUnusedMappers?: boolean;
+  /**
+   * @name noSchemaStitching
+   * @type boolean
+   * @description Disables Schema Stitching support
+   * @default false
+   * @warning The default behavior will be reversed in the next major release. Support for Schema Stitching will be disabled by default.
```

Comment:
@dotansimha I added `@warning` but we can rename it. It's just to provide more info in documentation (like deprecation and related).

It turns `@warning bla bla bla` into:

> bla bla bla